src/neynar-api/v2/openapi-farcaster/models/channel-or-dehydrated-channel.ts

```diff
@@ -0,0 +1,35 @@
+/* tslint:disable */
+/* eslint-disable */
+/**
+ * Farcaster API V2
+ * No description provided (generated by Openapi Generator https://github.com/openapitools/openapi-generator)
+ *
```

Comment:
this is relevant @tybook

Reply:
Yeah I figured out the relevant additions with Shreyas' help